install/kubernetes/helm/meshery/charts/meshery-operator/templates/clusterrole.yaml

```diff
@@ -30,7 +30,14 @@ rules:
   - patch
   - update
   - watch
-  - approve
+- apiGroups:
+  - meshery.layer5.io
+  resources:
+  - brokers/status
+  verbs:
```

Comment:
Please give any code comment for apiGroups' `verbs` if could what I mean we should sure which permissions we need to get.

Reply:
Also, should the templates/NOTES.txt be modified to capture the recent changes?

Reply:
@Aisuko @hexxdump TBH, I am not familiar enough with the system to give a good explanation on why we need those permissions for the cluster role, I made the changes just based on the plain yaml manifests that we're currently using... reference: https://github.com/meshery/meshery-operator/blob/master/config/manifests/default.yaml#L273-L288

Reply:
Good discussion. Given that the scope of these verbs is restricted to `meshery.layer5.io` resource groups, we shouldn't see any pushback from users given that the Meshery Operator should have full control over the lifecycle of its custom controllers.